pdns/recursordist/docs/settings.rst

```diff
@@ -946,6 +946,17 @@ Query example (where 192.0.2.14 is your server):
 PowerDNS can change its user and group id after binding to its socket.
 Can be used for better :doc:`security <security>`.
 
+.. _setting-signature-inception-skew:
+
+``signature-inception-skew``
+----------------------------------
+.. versionadded:: 4.1.5
+
+-  Integer
+-  Default: 60
+
+Allow the signture inception to be off by this number of seconds. Negative values are not allowed.
```

Comment:
signature -> signature

maybe: off -> early

Reply:
I think that something like "in the future" would be more clear than "early" (which at least seems confusing to me, but that could be a matter of perspective).